### DISCUSSION TITLE
DateFormat

### DISCUSSION THREAD

**Discussion by mihail-cornev** [totumonline/totum-ru-issues-and-discussions#424] [Как это сделать?] [answered]
Подскажите пожалуйста где я мог ошибиться?

тип поля - дата
пишу Код:
`=: dateFormat(date: $#nd; format: "Y")`
Но оно вместо года вставляет значение 01.01.1970

А если поставить типа поля - число, то работает нормально
В чём может быть проблема?

**Comment by erpic-ru** [comment, marked as answer]
Это же поле "дата" и у него есть формат y-m-d. Сделайте настройку "Формат вывода в интерфейс даты/времени" - "Y" и сможете видеть и выбирать только год, но в базе будет храниться текущая дата. Если нужно значение. 2023-01-01, то так и пропишите в формате "Y-01-01". Чтобы хранить 2023 нужно использовать поле строка, число или данные

**Reply by mihail-cornev** [reply]
Спасибо, а то не работал с тотум какое-то время и забыл простые вещи.
